[
  {
    "project": "arviz-devs/arviz",
    "commit": "8cb60f7ca1d0c5ff160cb6cf5ec6196e50afb97b",
    "filename": "arviz/stats/stats.py",
    "min_patch_found": false,
    "full_warning_msg": "arviz/stats/stats.py:1169:15 Unsupported operand [58]: `in` is not supported for right operand type `arviz.data.inference_data.InferenceData`.",
    "exception": "PatchSet should only have 1 Patch, but it has 0",
    "dd_fail": true
  },
  {
    "project": "arviz-devs/arviz",
    "commit": "8cb60f7ca1d0c5ff160cb6cf5ec6196e50afb97b",
    "filename": "arviz/stats/stats.py",
    "min_patch_found": false,
    "full_warning_msg": "arviz/stats/stats.py:1171:17 Unsupported operand [58]: `in` is not supported for right operand type `arviz.data.inference_data.InferenceData`.",
    "exception": "PatchSet should only have 1 Patch, but it has 0",
    "dd_fail": true
  }
]